lib/sms_autofill.dart

```diff
@@ -208,7 +210,7 @@ class _PhoneFieldHintState extends State<PhoneFieldHint> {
           focusNode: _focusNode,
           inputFormatters: _inputFormatters,
           decoration: InputDecoration(
```

Comment:
might be better to allow custom decoration instead no ? at least people would be able to customize the input as they want